projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -700,35 +1089,39 @@ export class OwcJsonService {
     }
   }
 
+  /**
+   * TODO: add more vendor params ??
```

Comment:
Here it was requested to add more vendor params e.g. like cql_filter so the data can be filtered by configuration.
See ukis-frontend/project-zki-viewer/-/issues/93

Reply:
+1 for this quest. Currently i put those directly in the request URL...